runtime/doc/treesitter.txt

```diff
@@ -180,6 +196,9 @@ TSNode:descendant_for_range({start_row}, {start_col}, {end_row}, {end_col})
 TSNode:named_descendant_for_range({start_row}, {start_col}, {end_row}, {end_col})
     Get the smallest named node within this node that spans the given range of
     (row, column) positions
+                                                        *TSNode:equal()*
+TSNode:equal({node})
+    Check if two nodes are identical.
```

Comment:
"Identical" (`===`) or "equal" (`==`)?

Reply:
🤷  this is passed directly to treesitter and the text here was copied from `treesitter/api.h`. I'm hoping it is `==` as opposed `===` since I think `===` is just pointer equality.

Reply:
Might be good to be a bit more explicit here -- using two different words (with customarily different meaning) can be confusing. I'll do a bit of research and make a proposal.

Reply:
How about
```
Check if {node} is part of the same tree and has the same id.
```
https://github.com/tree-sitter/tree-sitter/blob/c51896d32dcc11a38e41f36e3deb1a6a9c4f4b14/lib/src/node.c#L430-L432

Reply:
That sounds too technical. The point is they refer to _the same_ node within the same tree. checking the id is just the efficient way to verify that.

Reply:
Dam, this makes it basically useless. I'll keep it in, but I don't have a use for this. Thanks for looking into it.

Reply:
> they refer to the same node within the same tree. 

Yes, that is a better way of putting it here.